peakinvestigator/internal/progress.py

```diff
@@ -0,0 +1,25 @@
+## -*- coding: utf-8 -*-
```

Comment:
These aren't internal classes, but should be used when creating a custom progress. Please rename/move to `peakinvestigator.progress` package.